packages/cep47-client/README.md

```diff
@@ -2,26 +2,31 @@
 
 ## Installation
 
-`npm i casper-cep47-js-client`
+Run this command to install the CEP-47 JavaScript client:
+
+```
+npm i casper-cep47-js-client
+```
 
 ## Usage
 
-`casper-cep47-js-client` gives you and easy way to install and interact with Casper `cep47` smart contract.
+This client gives you an easy way to install and interact with the Casper CEP-47 (NFT) smart contract.
 
 ### Methods
```

Comment:
@hoffmannjan did you want to add these methods before merging? Or, you could merge this content, which looks good and add them later.